src/style.css

```diff
@@ -1 +1,94 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir, Arial, sans-serif;
```

Comment:
The `font-family` property should include a fallback font list, but it should not mix different font faces. Instead, declare the font family once and use the `font-weight` property to select the correct font face.